### ISSUE TITLE
Section name of Header synopsis

### ISSUE THREAD

**Issue by hewillk** [cplusplus/draft#6082]
For "Header `<meow>` synopsis", the section name in the draft has three styles
1. [xxx.syn]: Most common, such as  [string.syn].
2. [xxx.synop]: Rare, such as [any.synop].
3. [xxx..synopsis]: Verbose, seems to only appear with [iterator.synopsis].

Is it worthwhile to unify the section names of all headers into the first style?

**Comment by JohelEGP** [comment]
See https://github.com/cplusplus/draft/issues/1958#issuecomment-395513772.

**Comment by jensmaurer** [comment]
Additional note: xxx.syn can be auto-generated with simpler-to-use macros for references etc, thus is the preferred naming convention for new headers.

**Comment by winspool** [comment]
I was searching the valid C++26 header names with `.syn]`, but missed some headers.

While looking for the missing headers, I stumbled also over `iterator.synopsis`
and collected some more unusual synopsis tags:
 * `any`: any.synop
 * `iterator`: iterator.synopsis
 * `numeric`: numeric.ops.overview
 * `random`: rand.synopsis
 * `typeindex`: type.index.synopsis


In addition, some section tags end with `.syn`, but the start of the tag does not match the header name:
 * `scoped_allocator`: allocator.adaptor.syn
 * `memory_resource`: mem.sys.syn
 * `map`: associative.map.syn
 * `set`: associative.set.syn
 * `typeindex`: type.index.syn
I stopped here (there are probably more header entries affected)

**Comment by tkoeppe** [comment]
That's right. The stable labels are historic, and stable, and don't necessarily satisfy any global invariants. We don't usually change stable labels, since that would defeat half of their purpose.
